andela/codepirates-ah-frontend
# #167313353: Setup redux

**What does this PR do?**

- Setup and connect redux to react app

**Description of Task to be completed?**
- Install necessary packages
- Create redux folder
- Create actions folder with basic action types and action function
- Create reducers folder with index.js file and basic reducer
- create store
- Wrap react project in Provider component from the react-redux package.

**How should this be manually tested?**

- Clone the repo
- checkout to ch-setup-redux-167313353
- run **npm install**
- run **npm run devserver**
- use chrome browser and go to inspect and check redux devtool

**Any background context you want to provide?**
none
**What are the relevant pivotal tracker stories?**
[#167313353](https://www.pivotaltracker.com/story/show/167313353)

**Screenshots (if appropriate)**
<img width="1440" alt="Screen Shot 2019-09-26 at 11 28 14" src="https://user-images.githubusercontent.com/45101716/65676819-d4943900-e050-11e9-9d50-cb2b0948b7f7.png">
